src/Knp/Menu/Resources/views/knp_menu.html.twig

```diff
@@ -55,7 +55,8 @@
     {%- if matcher.isCurrent(item) %}
         {%- set classes = classes|merge([options.currentClass]) %}
     {%- elseif matcher.isAncestor(item, options.matchingDepth) %}
-        {%- set classes = classes|merge([options.ancestorClass]) %}
+        {# ancestorClass will be deprecated in future, use ancestor_class #}
```

Comment:
this does not report the deprecation AFAICT.

Reply:
Done in https://github.com/KnpLabs/KnpMenu/pull/351/commits/24fa9c2819d62945add341b3b5fdfc3cc7e72678